staging/operator-lifecycle-manager/deploy/chart/values.yaml

```diff
@@ -2,8 +2,12 @@ rbacApiVersion: rbac.authorization.k8s.io
 namespace: operator-lifecycle-manager
 # see https://kubernetes.io/docs/concepts/security/pod-security-admission/ for more details
 namespace_psa: 
-  enforceLevel: restricted
+  enforceLevel: baseline
   enforceVersion: latest
+  auditLevel: restricted
+  auditVersion: latest
+  warnLevel: restricted
+  warnVersion: latest
```

Comment:
are we sure we want latest here? there was a reason we pinned the version @anik120 do you remember it?

Reply:
These are the upstream chart values, we actually unset the warn and audit bits [here](https://github.com/openshift/operator-framework-olm/pull/420/files#diff-e8033c7c4bf9c1999d5b66faebd942551d51544ef15dd0d43f7efca074981684R411-R412) as the namespaces are restricted.